public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -336,47 +338,47 @@ include ../../../../_includes/_util-fns
 
 
   What if it had a dependency? What if it reported its activities through a logging service?
-  We'd apply the same *constructor injection* pattern, 
-  adding a constructor that takes a `logger` parameter.
+  We'd apply the same *constructor injection* pattern,
+  adding a constructor that takes a `Logger` parameter.
 
   Here is the revision compared to the original.
-  
+
 +makeTabs(
-  `dependency-injection/ts/app/heroes/hero.service.2.ts, 
+  `dependency-injection/ts/app/heroes/hero.service.2.ts,
   dependency-injection/ts/app/heroes/hero.service.1.ts`,
-  null, 
+  null,
   `app/heroes/hero.service (v.2),
   app/heroes/hero.service (v.1)`)
 
 :marked
   The constructor now asks for an injected instance of a `Logger` and stores it in a private property called `_logger`.
-  We call that property within our `getHeroes()` method when anyone asks for heroes.
+  We call that property within our `getHeroes` method when anyone asks for heroes.
 
   <a id="injectable"></a>
-  ### Why *@Injectable*?
+  ### Why @Injectable?
```

Comment:
I have no taste, but I've heard from people with taste that italics with boldface is a design no-no. Also, I've heard from real live readers who have trouble reading italics, so I tend to avoid them (italics, not readers).